packages/nodejs/src/__tests__/config.test.ts

```diff
@@ -27,6 +27,10 @@ describe("Configuration", () => {
     ignoreActions: [],
     ignoreErrors: [],
     ignoreNamespaces: [],
+    instrumentHttp: true,
+    instrumentHttps: true,
```

Comment:
```suggestion
```

Since the HTTP and HTTPS settings have been merged into `instrumentHttp`, the `instrumentHttps` entry can be removed from this map.